src/utils/color.js

```diff
@@ -0,0 +1,33 @@
+import { sha1 } from 'js-sha1';
+import { Hsluv } from 'hsluv';
```

Comment:
To keep the build small and avoid loading unwanted code (e.g. if `colorize_nicknames` is `false`), I think we should see if this can be loaded dynamically.

See for example here: https://github.com/conversejs/converse.js/blob/d4dfc3785b962973c8e17de9fa8f3ee7204329bc/src/headless/plugins/emoji/index.js#L80

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import

Hopefully it just works and Webpack should know what to do (i.e. create a separate file for `Hsluv` in `./dist`).

Reply:
oh, great! Will fix. I just have to check if making the `colorize` function async is possible.